pkg/services/accesscontrol/acimpl/service.go

```diff
@@ -237,6 +264,8 @@ func (s *Service) DeclarePluginRoles(ctx context.Context, ID, name string, regs
 // SearchUsersPermissions returns all users' permissions filtered by action prefixes
 func (s *Service) SearchUsersPermissions(ctx context.Context, usr identity.Requester,
 	options accesscontrol.SearchOptions) (map[int64][]accesscontrol.Permission, error) {
+	// Limit roles to available in OSS
+	options.RolePrefixes = []string{accesscontrol.BasicRolePrefix, accesscontrol.ManagedRolePrefix, accesscontrol.ExternalServiceRolePrefix}
```

Comment:
Enterprise reuses the same `SearchUsersPermissions()` store method, so we cannot put filters there. So instead I added it to the service method, to make sure we only return permissions available in OSS. Enterprise method does not contain filters.